src/printer.js

```diff
@@ -656,6 +656,7 @@ function genericPrintNoParens(path, options, print) {
                   join(concat([ ",", line ]), path.map(print, "elements"))
                 ])
               ),
+              n.elements[n.elements.length - 1] === null ? "," : "",
```

Comment:
I had no idea `[,]` produces an AST with an `elements` of length 1 as null, that's so weird. I forgot about this last night, but you can use the `util.getLast` to get the last element:

```js
util.getLast(n.elements) === null ? "," :""
```

This is a little confusing because `getLast` will return null if the array is empty, so make sure in your comment you mention that we've already checked if it's empty so it's guaranteed to have an element.